spec/requests/api/v1/listings_spec.rb

```diff
@@ -130,24 +130,24 @@
     expect(json['ami'].length).to eq(9)
   end
 
-  it 'gets Lottery Preferences' do
-    VCR.use_cassette('listings/lottery-preferences') do
-      get '/api/v1/listings/lottery-preferences.json'
-    end
+  # it 'gets Lottery Preferences' do
```

Comment:
@chrisdolendo correct, this was an old endpoint that was never really properly implemented -- in fact that's one of the big changes coming down the pipe from Luke is figuring out how to query for Listing Lottery Preferences, so he said this salesforce endpoint was going to be deprecated.